apps/coscrad-frontend/src/utils/generic-components/presenters/multilingual-text-presenter.tsx

```diff
@@ -17,43 +18,30 @@ export const MultilingualTextPresenter = ({
 }: MultilingualTextPresenterProps): JSX.Element => {
     const { defaultLanguageCode } = useContext(ConfigurableContentContext);
 
-    const { items } = text;
+    const textItemWithDefaultLanguage = getTextItemByLanguageCode(text, defaultLanguageCode);
 
-    const textItemWithDefaultLanguage =
-        items?.find((item) => item.languageCode === defaultLanguageCode) || null;
+    const primaryMultilingualTextItem = isNullOrUndefined(textItemWithDefaultLanguage)
+        ? getOriginalTextItem(text)
+        : textItemWithDefaultLanguage;
 
-    const translations = items?.filter((items) => items.languageCode !== defaultLanguageCode);
+    const { languageCode: languageCodeOfPrimaryTextItem } = primaryMultilingualTextItem;
+
+    const translations = getTextItemsNotInLanguage(text, languageCodeOfPrimaryTextItem);
```

Comment:
I feel like `getTextItemsNotInLanguage` is the most direct description of this helper, but it would be nicer if it were called `getTranslations` or something like that.  It's just that it's nice to have a title that says exactly what it's doing to get the translations.